rest_server/main.go

```diff
@@ -0,0 +1,130 @@
+// Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
+// Use of this source code is governed by a license that can be found
+// in the LICENSE file.
+
+package main
+
+// specs from https://github.com/siodb/siodb/blob/master/docs/dev/designs/RestServer.md
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+var (
+	SiodbInstanceConfigurationRootPath string = "/etc/siodb/instances"
+	restServerConfig                   RestServerConfig
+	IOMgrCPool                         *IOMgrConnPool
+	IOMgrCPoolMinConn                  = 4
+	IOMgrCPoolMaxConn                  = 8
+	siodbLoggerPool                    *SiodbLoggerPool
+)
+
+func main() {
+
+	// Parse Args
+	SiodbInstanceName := flag.String("instance", "", "Instance name")
+	flag.Parse()
+	if len(*SiodbInstanceName) == 0 {
+		fmt.Printf("%v %v %v %v %s\n", time.Now().UTC().Format("2006-01-02 15:04:05.999999"),
+			SeverityLevelToString(FATAL), unix.Getpid(), unix.Gettid(), "Invalid instance name in argument '--instance'")
+		os.Exit(1)
+	}
+
+	// Signals
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+
+	// Variables
+	SiodbInstanceConfigurationPath := SiodbInstanceConfigurationRootPath + "/" + *SiodbInstanceName
+	var err error
+
+	// Parse Siodb instance parameters file
+	siodbConfigFile := &SiodbConfigFile{}
+	siodbConfigFile.path = SiodbInstanceConfigurationPath + "/config"
+	if err := siodbConfigFile.ParseParameters(); err != nil {
+		fmt.Printf("%v %v %v %v %s\n", time.Now().UTC().Format("2006-01-02 15:04:05.999999"),
+			SeverityLevelToString(FATAL), unix.Getpid(), unix.Gettid(), err)
+		os.Exit(2)
+	}
+
+	// Logging setup
+	if siodbLoggerPool, err = CreateSiodbLoggerPool(siodbConfigFile); err != nil {
+		fmt.Printf("%v %v %v %v %s\n", time.Now().UTC().Format("2006-01-02 15:04:05.999999"),
+			SeverityLevelToString(FATAL), unix.Getpid(), unix.Gettid(), err)
+		os.Exit(2)
+	}
+	siodbLoggerPool.ConfigGinLogger()
+	siodbLoggerPool.Info("Logging started")
+
+	// Create REST Server config
+	if err = restServerConfig.ParseAndValidateConfiguration(siodbConfigFile); err != nil {
+		siodbLoggerPool.FatalAndExit(FATAL_INIT_ERROR, "cannot parse the rest configuration properly", err)
+	}
+	siodbLoggerPool.Info("REST server config initialized successfully")
+
```

Comment:
Output here message like this:
https://github.com/siodb/siodb/blob/418943496152ec1bd0817e21da1b8a14b463f198/siodb/lib/Siodb.cpp#L130

Omit "Compiled on" part by now.
Version constants SIODB_VERSION_MAJOR, SIODB_VERSION_MINOR, SIODB_VERSION_PATCH - integers and SIODB_COPYRIGHT_YEARS (string)  should be placed in the new file called Version.go. Later on, when we integrate with whole siodb build, this fill will be generated.

Reply:
````
2020-09-30 12:44:08.841014 info 2265012 2265015 Logging started
2020-09-30 12:44:08.841126 info 2265012 2265015 Siodb REST Server v.0.8.0
2020-09-30 12:44:08.841149 info 2265012 2265015 Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
2020-09-30 12:44:08.842643 info 2265012 2265015 IOMgr connection pool initialized successfully
^C2020-09-30 12:44:52.961396 info 2265012 2265015 interrupt signal received, terminating...
2020-09-30 12:44:52.962386 info 2265012 2265015 IOMgr connection pool stopped.
2020-09-30 12:44:52.962549 info 2265012 2265015 Logging stopped.
```